content/posts/newsletter/2020-07-01.md

```diff
@@ -0,0 +1,163 @@
+---
+title: XMPP works - 1 July 2020
+save_as: 2020/07/newsletter-01-july/index.html
+url: 2020/07/newsletter-01-july/
+date: 2020-07-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of June 2020.
+
+## Subscribe to the newsletter
+
+We suggest you [Subscribe](https://tinyletter.com/xmpp) to receive the next editions in your inbox as soon as it is published! Promote this newsletter to whoever may be interested.
+
+## Help us to build the newsletter
+
+We started drafting every new issue in the [XSF Github repository](https://github.com/xsf/xmpp.org/pulls). 
+And we are always happy to welcome contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. 
+
+You have a project and (want to) write about it? Please consider sharing your news or events here, and promote it to a large audience! Even if you can only spend a few minutes, these would already be helpful!
+Tasks which need to be done on a regular basis are for example:
+- Aggregation of news in the XMPP universe
+- Short formulation of news and events
+- Summary of the monthly communication on extensions (XEP)
+- Review of the draft
+- Translations: especially German and Spanish
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- [French](https://linuxfr.org/tags/xmpp/public)
+
+## XSF Announcements
+
+JC Brand created an [XMPP job board](https://xmpp.work/)! This enables people to review job offers as well as advertise XMPP services. Please take a look, post your own offers and recommend it. XMPP works!
+
+## Article
+
+Ingo Jürgensmann wrote an article on the backround of the 'Jabber' and 'XMPP' terminology and their [differences and similarities](https://blog.windfluechter.net/content/blog/2020/06/09/1758-jabber-vs-xmpp).
+
+The XMPP client Dino is one out of three projects which were awarded with the [10th Thomas Krenn Award](https://www.thomas-krenn.com/de/unternehmen/presse/pressemeldung.thomas-krenn-award-2020.html) (German).
```

Comment:
This announce uses `(German)` at the end, whereas one below for french uses `:fr:` at the beginning